Sources/StreamChatUI/ChatMessageList/ChatMessage/StackViewDecoratorView.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// The decorator view that is used to stack multiple decorators.
```

Comment:
Can we add more context to this comment? It is not 100% clear why when need this stack of multiple decorators 🤔